base/sysinfo.jl

```diff
@@ -322,4 +324,83 @@ windows_version
 
 const WINDOWS_VISTA_VER = v"6.0"
 
+"""
+    Sys.isexecutable(path::String)
+
+Returns `true` if the given `path` has executable permissions.
```

Comment:
We word our docs in the imperative, i.e. this should be "return" rather than "returns." Same for the `which` docstring below.

Reply:
Hah, I was basing this off of the `Sys.windows_version()` docstring above it.  I'll fix that too.